docs/documentation/platform/dynamic-secrets/mongo-db.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: "Mongo DB"
+description: "Learn how to dynamically generate Mongo DB Database user credentials."
+---
+
+The Infisical Mongo DB dynamic secret allows you to generate Mongo DB Database credentials on demand based on configured role.
+
+
+<Info>
+				If your using Mongo Atlas, please use [Atlas Dynamic Secret](./mongo-atlas) as MongoDB commands are not supported by atlas.
+</Info>
+
+## Prerequisite
+Create a user with the required permission in your SQL instance. This user will be used to create new accounts on-demand.
+
+## Set up Dynamic Secrets with Mongo DB
+
+<Steps>
+  <Step title="Open Secret Overview Dashboard">
+	Open the Secret Overview dashboard and select the environment in which you would like to add a dynamic secret.
+  </Step>
+  <Step title="Click on the 'Add Dynamic Secret' button">
+	![Add Dynamic Secret Button](../../../images/platform/dynamic-secrets/add-dynamic-secret-button.png)
+  </Step>
+  <Step title="Select Mongo DB">
+	![Dynamic Secret Modal](../../../images/platform/dynamic-secrets/dynamic-secret-modal-mongodb.png)
+  </Step>
+  <Step title="Provide the inputs for dynamic secret parameters">
+	<ParamField path="Secret Name" type="string" required>
+		Name by which you want the secret to be referenced
```

Comment:
Nit: I see for some of the ParamFields we are ending each sentence with a dot, and others we are not. Let's try to keep it consistent, and add a dot (.) at the end of each sentence

Reply:
Good catch